src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -65,6 +66,7 @@ public DocumentDatabase(string name, RavenConfiguration configuration, ServerSto
             IndexStore = new IndexStore(this, _indexAndTransformerLocker);
             TransformerStore = new TransformerStore(this, _indexAndTransformerLocker);
             SqlReplicationLoader = new SqlReplicationLoader(this);
+            EtlLoader = new EtlLoader(this);
```

Comment:
Why do we have both `SqlReplicationLoader` and `EtlLoader`? Aren't they the same in this case?